cli/cache/mod.rs

```diff
@@ -87,21 +53,11 @@ impl Loader for FetchCacher {
   fn get_cache_info(&self, specifier: &ModuleSpecifier) -> Option<CacheInfo> {
     let local = self.file_fetcher.get_local_path(specifier)?;
     if local.is_file() {
-      let location = &self.disk_cache.location;
-      let emit = self
-        .disk_cache
-        .get_cache_filename_with_extension(specifier, "js")
-        .map(|p| location.join(p))
-        .filter(|p| p.is_file());
-      let map = self
-        .disk_cache
-        .get_cache_filename_with_extension(specifier, "js.map")
-        .map(|p| location.join(p))
-        .filter(|p| p.is_file());
       Some(CacheInfo {
         local: Some(local),
-        emit,
-        map,
+        // we no longer store the emit and map in their own files
```

Comment:
Yes, see the bottom of the PR description. It was agreed upon that breaking this was worth it in order to fix the issues.

That said, an alternative to not be breaking is to store a hash of the emit in addition to the source in the current version file. Then we would load the source, versions (two hashes), and emit then check that the emit hash matches the one in versions and the source hash matches the one in versions. If they do, then we can use the emit. By breaking here the main gain we get is that we don't need to bother hashing the emit... the more I think about it the more I would prefer this instead actually and it would be less work, though still breaking for deno_cache. Thoughts?

Reply:
It would be really easy to pivot this PR to that actually. I think I will try it out on Monday and see how it performs. I'm getting a little concerned depending on sqlite so much.

Reply:
I got a bit carried away and opened this PR that does this: https://github.com/denoland/deno/pull/15220 -- I think it's better